canonical-web-and-design/microstack.run
# Rebuild MicroStack home page

## Done

Rebuild MicroStack home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit) and and resolve comments
- Check against [design](https://app.zeplin.io/project/611e22af1491325814021275/screen/611e22e82bc2085acc4bc1df)

## Issue / Card

Fixes [#4351](https://github.com/canonical-web-and-design/web-squad/issues/4351)

## Screenshots
<img width="1440" alt="MicroStack - OpenStack in a snap ()" src="https://user-images.githubusercontent.com/57550290/131124842-bcd602cb-211d-40c8-8918-54e42a3436ef.png">
